cypress/integration/send_funds.spec.js

```diff
@@ -0,0 +1,140 @@
+before(() => {
+  cy.nukeAllState();
+
+  cy.createIdentity("coolname");
+  cy.registerUser("coolname");
+  cy.registerOrg("coolorg");
+});
+
+context("navigation", () => {
+  beforeEach(() => {
+    cy.visit("public/index.html");
+  });
+  it("opens the modal from the user wallet", () => {
+    // user wallet opens modal
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").should("exist");
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+  });
+  it("opens the modal from the org wallet", () => {
+    // org wallet opens modal
+    cy.pick("sidebar", "org-coolorg").click();
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").should("exist");
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+  });
+  it("closes the modal by clicking X", () => {
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").should("exist");
+    cy.pick("amount-input").type("123");
```

Comment:
We don't need the input checks here, as we only test closing the modal.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/712/commits/0f58c678731a91801d6d0bc6f01096cc72045ef3